### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by bigbangk123** [aiko-chan-ai/DiscordBotClient#20] [General]
hey  bug 
![image](https://user-images.githubusercontent.com/99186742/232408045-41ea8f20-4312-44e9-a6e7-c7ffc76b1065.png)
![image](https://user-images.githubusercontent.com/99186742/232408056-b1d0ff18-5660-4909-84d5-908dddc5ece6.png)
![image](https://user-images.githubusercontent.com/99186742/232408062-49aa22c4-f352-4845-9f30-a41e046f8dc6.png)
![image](https://user-images.githubusercontent.com/99186742/232408074-d471ac98-374e-45c1-8048-949a603330c7.png)

**Comment by aiko-chan-ai** [comment]
you should only use the necessary intents

**Reply by bigbangk123** [reply]
How do I fix it?

**Reply by aiko-chan-ai** [reply]
https://github.com/aiko-chan-ai/DiscordBotClient#download--releases

**Reply by dkashdf** [reply]
I can't exit the Token Bot

**Reply by aiko-chan-ai** [reply]
Huh

**Reply by bigbangk123** [reply]
How to fix

**Reply by bigbangk123** [reply]
I can't exit the Token Bot

**Reply by aiko-chan-ai** [reply]
do you mean "logout"

**Reply by bigbangk123** [reply]
yes

**Reply by aiko-chan-ai** [reply]
Same as user (setting > logout)

**Reply by bigbangk123** [reply]
https://streamable.com/b6a44v

**Reply by aiko-chan-ai** [reply]
https://streamable.com/xf4xhh

**Reply by bigbangk123** [reply]
I'm accessing someone else's bot token without permission. and i can't do it

**Reply by aiko-chan-ai** [reply]
dont use two privileged intents `GUILD_PRESENCES` and `GUILD_MEMBERS`
If the bot still doesn't work, it's because the bot doesn't have `MESSAGE_CONTENT`

**Reply by bigbangk123** [reply]
What do I have to do?

**Reply by aiko-chan-ai** [reply]
...

**Reply by dkashdf** [reply]
I can't open anything because it's someone else's bot token.

**Reply by NekoCyan** [reply]
> I can't open anything because it's someone else's bot token.

So just stop steal token, be good.
